perf/datarepo-api.yaml

```diff
@@ -10,7 +10,10 @@ env:
   DB_STAIRWAY_URI: jdbc:postgresql://perf-jade-gcloud-sqlproxy.perf:5432/stairway
   DB_DATAREPO_URI: jdbc:postgresql://perf-jade-gcloud-sqlproxy.perf:5432/datarepo
   IT_JADE_API_URL: https://jade-perf.datarepo-perf.broadinstitute.org
-  DATAREPO_GCS_ALLOWREUSEEXISTINGBUCKETS: true
+  DATAREPO_RESOURCEID: broad-jade-perf
+  SAM_STEWARDSGROUPEMAIL: JadeStewards-perf@dev.test.firecloud.org
```

Comment:
I'm not totally confident that these belong here. I would think we would want to the resourceId to be specific to a project.